packages/mui-codemod/src/v6.0.0/system-props/removeSystemProps.js

```diff
@@ -149,7 +149,9 @@ export default function removeSystemProps(file, api, options) {
       matcher: (key, val) =>
         key !== 'color' ||
         (val.value?.includes('.') && val.value !== 'inherit') ||
-        val.value === 'divider',
+        val.value === 'divider' ||
+        val.value.startsWith('#') ||
+        val.value.match(/\(.*\)/),
```

Comment:
What's this one for?

Reply:
`val.value.startsWith('#')` is for random hex color `#ff5252`
`val.value.match(/\(.*\)/)` is for color with bracket like `rgba(….)` or `hsl(…)`

The above cases should be moved to `sx` prop.